app/commands.py

```diff
@@ -84,3 +85,26 @@ def post_tweet_command():
         exit(0)
     else:
         exit(1)
+
+
+@app.cli.command('cleanup-db')
```

Comment:
`clean db` would be better name for a command :)

Reply:
Changed